src/style.css

```diff
@@ -0,0 +1,77 @@
+.boxed {
+  
+  border: 5px solid #9fccfc ;
+  border-radius: 50px;
+ background-color: #eaf4ff;
```

Comment:
If you want the border to not be so close to the edge of the place, then you just need to put `margin: 10px` here, it will create a 10 pixel margin (or replace that with whatever size you need) between the `"boxex"` div and the `body` element.

[Further reading](https://www.w3schools.com/css/css_margin.asp)